apps/typed-dependency-tester/README.md

```diff
@@ -0,0 +1,17 @@
+# Typed Dependency Tester
+
+The Typed Depndency Tester is a tool used to detect if types in `MicrosoftTeams.d.ts` are importing types from other modules and giving them an implicit `any` resolution
+
+## Getting Started
+
+### Running the Dependency Tester
+
+If you would like to use this tester locally, simply type `pnpm build` in your terminal. The tester will then proceed to install the workspace version of the TeamsJS.
+
+If you would like to test against a specific version of Teams JS you will need to edit the `@microsoft/teams-js` version in the `package.json` file.
```

Comment:
`Teams JS` naming on this line should match whatever we use elsewhere in the README